tests/lua/test.sh

```diff
@@ -5,5 +5,5 @@ SCRIPT_DIR="$(cd "$(dirname "$0" )" && pwd)"
 
 ulimit -s 16384 # debug build requires this much stack to pass tests
```

Comment:
Should we increase this limit instead?

Reply:
Running in release mode seems fine. We could also run in release with debug info mode if we want.